lib/gooddata/models/user_filters/user_filter_builder.rb

```diff
@@ -423,11 +426,55 @@ def self.execute_mufs(user_filters, options = {})
                            end
                          end
                        end
+
+      log(delete_results, options[:project], to_delete) if GoodData.logging_on?
+
       { created: to_create, deleted: to_delete, results: create_results + (delete_results || []) }
     end
 
     private
 
+    # Log adding/removing user-filters results
+    # @param results [Array] list of status objects, e.g: { status => :successful || :failed, user: user_profile_url, type: :delete || :create }
+    # @param project [Project] project object
+    # @param user_filters [Array] list of MandatoryUserFilter objects
+    # @return nil
+    def self.log(results, project, user_filters)
+      profile_url_to_login = nil
+      log_success_create_result_proc = proc do |readable_user_login, filter_uris|
+        GoodData.logger.info "Created user-filter=#{filter_uris} for user=#{readable_user_login} in project=#{project.pid}"
+      end
+      log_success_delete_result_proc = proc do |readable_user_login, filter_uris|
+        GoodData.logger.warn "Deleted user-filter=#{filter_uris} of user=#{readable_user_login} in project=#{project.pid}"
```

Comment:
For disabling/removing operator, log level is set to WARN so that user can intent on it.